fixtures/flight/config/webpack.config.js

```diff
@@ -199,7 +199,7 @@ module.exports = function (webpackEnv) {
       ? shouldUseSourceMap
         ? 'source-map'
         : false
-      : isEnvDevelopment && 'cheap-module-source-map',
+      : isEnvDevelopment && 'source-map',
```

Comment:
Is this a requirement for this feature to work?

Reply:
No, it's just for client source maps. They end up off by one and not mapping to the right columns so not testing the proper experience.